sys/phydat/phydat_str.c

```diff
@@ -64,6 +64,8 @@ const char *phydat_unit_to_str(uint8_t unit)
         case UNIT_TEMP_K:   return "K";
         case UNIT_LUX:      return "lx";
         case UNIT_M:        return "m";
+        case UNIT_M2:       return "sqm";
+        case UNIT_M3:       return "cbm";
```

Comment:
My alternative has the drowback btw if someone for some weird reason decides to have e.g. 10m² returned as { { 1 }, UNIT_M2, 1 }. Then my implementation would return it as `1sqm`.